index.html

```diff
@@ -0,0 +1,281 @@
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css"
+    />
+    <link rel="stylesheet" href="https://nebulaui.netlify.app/CSS/global.css" />
+    <link rel="stylesheet" href="style.css" />
+    <title>SilkShopy devlopment</title>
+  </head>
+  <body>
+    <!--------------.------- nav bar ---------------------------->
+    <header class="header-shopy shadow-box">
+      <nav class="nav-shopy flex-center">
+        <section class="brand-name">
+          <img class="silk-img" src="/images/silkshopy-name.jpeg" alt="" />
+        </section>
+
+        <div class="nav-items flex-center">
+          <section class="nav-input-box flex-center">
+            <button class="btn-input-box">
+              <i class="fas fa-search"></i>
+            </button>
+            <input class="input-shopy" type="text" placeholder="search" />
+          </section>
+          <section class="nav-icons flex-center">
+            <button class="btn btn-login btn-primary">Login</button>
+
+            <div class="nav-icons-item flex-center">
+              <span class="icon-shopy"><i class="fas fa-heart"></i></span>
+              <span class="below-text">Wishlist</span>
+            </div>
+
+            <div class="nav-icons-item flex-center">
+              <span class="icon-shopy"
+                ><i class="fas fa-shopping-cart"></i
+              ></span>
+              <span class="below-text">Cart</span>
+            </div>
+          </section>
+        </div>
+      </nav>
+    </header>
+
+    <!--------------------- banner image ------------------------->
+    <section class="banner-image-container">
+      <img src="/images/redmi-9a.jpg" class="banner-image" />
+      <button class="btn btn-image btn-primary">Shop Now</button>
+    </section>
+
+    <!--------------------- show logos ---------------------------->
+    <section class="show-brand-logos">
+      <img src="/images/apple.png" class="brand-logos-img" />
+      <img src="/images/mi.png" class="brand-logos-img" />
+      <img src="/images/realme.png" class="brand-logos-img contain" />
+      <img src="/images/samsung.png" class="brand-logos-img" />
+      <img src="/images/vivo.png" class="brand-logos-img" />
+      <img src="/images/motorola.png" class="brand-logos-img" />
+      <img src="/images/sony.png" class="brand-logos-img" />
+      <img src="/images/asus.png" class="brand-logos-img" />
+    </section>
+
+    <section class="features-heading-box flex-center">
+      <h1 class="feature-heading">Featured Categories</h1>
+    </section>
+
+    <div class="title-underline"></div>
+
+    <div class="feature-grid-layout">
+      <section class="feature-cat-container">
+        <div class="img-box">
+          <img src="/images/back-smartp-img.jpeg" class="back-imgs" />
+        </div>
+        <div class="text-overlay flex-center">Smart Phones</div>
+      </section>
```

Comment:
Here a suggestion would be that you could give the outer `div` i.e. the one having class `feature-grid-layout` a tag of section and then the inner children i.e. the one having classname of `feature-cat-container` could be given a tag of `article`. This helps in maintaining consistency as the upper elements have section to the wrapper itself.

e.g. 
```HTML
<section class="feature-grid-layout">
    <article class="feature-cat-container">
        <figure class="img-box">
            <img src="/images/back-smartp-img.jpeg" class="back-imgs" />
        </figure>
        <figcaption class="text-overlay flex-center">Smart Phones</figcaption>
    </article>
    ...
</section>
```

You could do this throughout the project for all similar elements.